src/qt/guiutil.cpp

```diff
@@ -272,6 +272,12 @@ bool hasEntryData(const QAbstractItemView *view, int column, int role)
     return !selection.at(0).data(role).toString().isEmpty();
 }
 
+void loadFont(const QString& file_name)
```

Comment:
Why not return `id` for generality?

From Qt [docs](https://doc.qt.io/qt-5.12/qfontdatabase.html#addApplicationFont):
>  An ID is returned that can be used to remove the font again with `removeApplicationFont()` or to retrieve the list of family names contained in the font.